web/src/markdown.ts

```diff
@@ -70,7 +118,10 @@ export function translate_emoticons_to_names({src, get_emoticon_translations}) {
     return translated;
 }
 
-function contains_problematic_linkifier({content, get_linkifier_map}) {
```

Comment:
Would changing the parameters to be not an object be better as a prep commit?

Reply:
yes